src/endWindow.java

```diff
@@ -15,6 +15,7 @@
      */
     public endWindow() {
         initComponents();
+int a=0;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "a" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1481)